tests/integration_tests/functional/test_rng.py

```diff
@@ -7,8 +7,11 @@
 from framework.properties import global_props
 from framework.utils import check_entropy
 
-if global_props.instance == "c7g.metal" and global_props.host_linux_version == "4.14":
-    pytestmark = pytest.mark.skip(reason="c7g requires no SVE 5.10 kernel")
+if (
+    global_props.cpu_model == "ARM_NEOVERSE_V1"
+    and global_props.host_linux_version == "4.14"
+):
+    pytestmark = pytest.mark.skip(reason="[cm]7g requires no SVE 5.10 kernel")
```

Comment:
Would it be more explicit to say
```suggestion
    pytestmark = pytest.mark.skip(reason="Graviton 3 requires no SVE 5.10 kernel")
```